web/src/features/PreflightChecks/types.ts

```diff
@@ -0,0 +1,67 @@
+/* REQUEST TYPES */
```

Comment:
dunno what these are yet. Also I copied some of the type definitions you did a while back. I will refactor them out of the main types file when I update the components.